binder.go

```diff
@@ -19,12 +19,13 @@ type NoUnknownFieldsBinder struct{}
 	`DisallowUnknownFields` Decoder setting for unmarshaling json.
 */
 func (binder *NoUnknownFieldsBinder) Bind(i interface{}, c echo.Context) error {
-	// if there is no body on the request - return early
-	if c.Request().Body == http.NoBody || c.Request().Body == nil {
-		return util.NewErrBadRequest("no body")
-	}
```

Comment:
Removing this check makes me nervous but I couldn't get it to trigger - so it's probably fine!